npm-shrinkwrap.json

```diff
@@ -1,1434 +1,1209 @@
 {
   "name": "@okta/okta-signin-widget",
-  "version": "1.3.1",
+  "version": "1.3.2",
   "dependencies": {
     "@okta/i18n": {
-      "version": "1.3.0-beta.3",
-      "from": "@okta/i18n@1.3.0-beta.3"
+      "version": "1.11.0"
+    },
+    "@okta/install-with-shrinkwrap": {
+      "version": "1.2.0",
+      "dependencies": {
+        "ansi-regex": {
```

Comment:
Can you verify what the behavior is here? Main worry is that we're going to include subdependencies when we check in our node_modules/@okta folder (i.e. it looks like these dependencies would be in the node_modules folder of install-with-shrinkwrap).